modules/programs/gh.nix

```
@@ -135,15 +135,16 @@ in {
     #
     # See https://github.com/nix-community/home-manager/issues/4744 for details.
     home.activation.migrateGhAccounts =
-      hm.dag.entryBetween [ "linkGeneration" ] [ "writeBoundary" ] ''
-        if [[ -e "${config.xdg.configHome}/gh/hosts.yml" ]]; then
+      let ghHosts = "${config.xdg.configHome}/gh/hosts.yml";
+      in hm.dag.entryBetween [ "linkGeneration" ] [ "writeBoundary" ] ''
+        if [[ ! -L "${ghHosts}" ]] && [[ -f "${ghHosts}" ]]; then
```

No need for separate `[[` blocks.
```suggestion
        if [[ ! -L "${ghHosts}" && -f "${ghHosts}" ]]; then
```

---

Btw, `-f` should imply `! -L`. I.e., it should be sufficient with

```suggestion
        if [[ -f "${ghHosts}" ]]; then
```

---

> Btw, `-f` should imply `! -L`. I.e., it should be sufficient with

That was also my first attempt. However, for some reason it does not behave as expected.

---

Ah, odd. Anyway, fair enough to keep it.